plugins/verifier/sbom/utils/spdxutils.go

```diff
@@ -31,3 +33,26 @@ func GetPackageLicenses(doc spdx.Document) []PackageLicense {
 	}
 	return output
 }
+
+// returns true if the licenseExpression contains the disallowed license
+// this implements a whole word match
+func ContainsLicense(spdxLicenseExpression string, disallowed string) bool {
+	if len(spdxLicenseExpression) == 0 {
+		return false
+	}
+
+	// if the licenseExpression is exactly the same as the disallowed license, return true
+	if spdxLicenseExpression == disallowed {
+		return true
+	}
+
+	disallowed1 := disallowed + " "
+	disallowed2 := " " + disallowed
+
+	// look for whole word match
+	if strings.Contains(spdxLicenseExpression, disallowed1) || strings.Contains(spdxLicenseExpression, disallowed2) {
```

Comment:
nit: do you think regex would be useful here?

Reply:
thanks! i ve tried a simple regex that mathes on word boundarie, \b(MIT)\b. However this fails the cases like MIT-2.0, because - is by default a word boundary.

Reply:
I can do more research on more complex expressions, maybe something like (?:^|\s|$)(MIT)(?:^|\s|$) this will work.. will need to do more testing.

Reply:
Sure we can do a follow up PR after?